src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -227,9 +233,31 @@ private function getDocBlockFromMethod(string $class, string $ucFirstProperty, i
         }
 
         try {
-            return [$this->docBlockFactory->create($reflectionMethod, $this->contextFactory->createFromReflector($reflectionMethod)), $prefix];
+            return [$this->docBlockFactory->create($reflectionMethod, $this->createFromReflector($reflectionMethod)), $prefix];
```

Comment:
Here the `Context` passed to `create()` is now no longer based on the method, but on the class. Does it still work as expected?

Reply:
this was already the case in the implementation `contextFactory->createFromReflector`